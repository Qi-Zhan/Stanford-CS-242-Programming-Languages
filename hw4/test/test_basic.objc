START VERIFY;
# Test free variables don't step.
x;
# Objects don't step.
x = [f1 = \o.o];
# Basic method invocation.
x.f1;
# Basic method override.
x.f2 <- \o.[];
STOP VERIFY;
