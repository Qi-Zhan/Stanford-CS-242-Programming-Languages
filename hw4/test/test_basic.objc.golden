x;
x = [f1 = \o.o];
[f1 = \o.o];
[f1 = \o.o, f2 = \o.[]];
